modules/hivemq-edge-module-etherip/src/main/java/com/hivemq/edge/adapters/etherip/EtherIpProtocolAdapterInformation.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2023-present HiveMQ GmbH
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.hivemq.edge.adapters.etherip;
+
+
+import com.hivemq.adapter.sdk.api.ProtocolAdapterCapability;
+import com.hivemq.adapter.sdk.api.ProtocolAdapterCategory;
+import com.hivemq.adapter.sdk.api.ProtocolAdapterInformation;
+import com.hivemq.adapter.sdk.api.ProtocolAdapterTag;
+import org.apache.commons.io.IOUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.EnumSet;
+import java.util.List;
+
+public class EtherIpProtocolAdapterInformation implements ProtocolAdapterInformation {
+
+    public static final ProtocolAdapterInformation INSTANCE = new EtherIpProtocolAdapterInformation();
+    private static final @NotNull Logger log = LoggerFactory.getLogger(EtherIpProtocolAdapterInformation.class);
+
+    protected EtherIpProtocolAdapterInformation() {
+    }
+
+    @Override
+    public @NotNull String getProtocolName() {
+        return "Ethernet/IP CIP 2";
```

Comment:
I propose we replace the old protocol adapter by the new one and remove the suffx "2" from name and id.

Reply:
This also means we remove the old protocol adapter from the code in order to not have duplicates.

Reply:
Old adapter has been removed.
Names have bee nadjusted.